<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chain_coding</class>
 <widget class="QDialog" name="Chain_coding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>969</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chain Coding</string>
  </property>
  <widget class="QGroupBox" name="chainCode_inputs">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_chainCode_cannyLowerThres">
          <property name="text">
           <string>Lower Threshold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_chainCode_cannyLowerThres"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_chainCode_upperThres">
          <property name="text">
           <string>Upper Threshold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_chainCode_upperThres"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_chainCode_loadData">
        <property name="text">
         <string>Load Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_chainCode_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_chainCode_data">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>611</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>780</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_chainCode_Output">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_chainCode_Output"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_chainCode_Output">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
